src/libsystemd/libsystemd.sym

```diff
@@ -218,6 +218,7 @@ global:
         sd_bus_send_to;
         sd_bus_call;
         sd_bus_call_async;
+        sd_bus_reply;
```

Comment:
Nah, please add this line to v248.

Reply:
yikes, fixed, didn't look closely enough to see this was ordered per version